### DISCUSSION TITLE
Add search criteria Trace Message

### DISCUSSION THREAD

**Discussion by chexpirit** [rappen/PluginTraceViewer#170] [Ideas]
Hi, 

would be helpful to add a way to search by the Trace Message content.

Thanks

**Comment by rappen** [comment]
The "Quick Filter" can search (=filter) by the message, see here:

![image](https://user-images.githubusercontent.com/2572253/169827660-eab8830d-3f80-4e28-abe8-03b956a987d5.png)

This will filter all retrieved traces by this text "anywhere".

If you select the Trace Message and type <CTRL>+F the search dialog pops up and then try to select the text you wrote:

![image](https://user-images.githubusercontent.com/2572253/169828332-e3eee605-e4a6-4e57-845d-05eda3951c38.png)

Is this enough? You need more?

**Comment by chexpirit** [comment]
This looks great. 

Thanks!

**Comment by rappen** [comment]
Great!
Keep it up 😊
